src/quantum/azext_quantum/profiles.py

```diff
@@ -0,0 +1,10 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.core.profiles import CustomResourceType
+
+
+QUANTUM_DATA = CustomResourceType('azext_quantum.vendored_sdks.azure_quantum', 'QuantumClient')
```

Comment:
Is this an Azure CLI extension idiom? I'm not quite sure what these two lines do, since the values `QUANTUM_DATA` and `QUANTUM_MGMT` don't seem to be used anywhere.